sources/core/objects/MedbayTools.cpp

```diff
@@ -0,0 +1,60 @@
+#include "MedbayTools.h"
+
+HealthAnalyzer::HealthAnalyzer(quint32 id) : Item(id)
+{
+    SetSprite("icons/device.dmi");
+    SetState("health");
+    name = "Health Analyzer";
+}
+
+void HealthAnalyzer::Scan(IdPtr<Human> target)
+{
+    float health = target->GetHealth() / 100.0f;
+    float suffocation_damage = target->GetSuffocationDamage() / 100.0f;
+    float burn = target->GetBurnDamage() / 100.0f;
+    float brute = target->GetBruteDamage() / 100.0f;
+
+    GetGame().GetChat().PostTextFor(QString("Analyzing results for %1:\nOverall status:").arg(target->name), GetOwner());
+
+    GetGame().GetChat().PostTextFor(target->IsDead() ? QString("Deceased") : QString("%1 healthy").arg(health), GetOwner());
+    if (target->GetBruteDamage() > 1000)
+    {
+        GetGame().GetChat().PostTextFor(QString("%1 tissue damage detected.").arg(target->GetBruteDamage() > 5000 ? "Severe" : "Minor"), GetOwner());
```

Comment:
https://en.wikipedia.org/wiki/Soft_tissue_injury, I can change it it's from tg medscanner.